### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by ayush-09**

### CODE REVIEW

**Comment by ayush-09** [comment]
Added me! Issue #325

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -532,5 +532,12 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/31401437/106563296-974b8f80-6551-11eb-8753-a13ea9e944c2.png",
     "issueNumber": 304
+  },
+  {
+    "name": "Ayush Varshney",
+    "githubUsername": "ayush-09",
+    "imageUrl": "https://user-images.githubusercontent.com/51924622/107047890-54f2ae80-67ee-11eb-8995-90c289eb8346.png",
+    "issueNumber": 325
   }
+
```
Comment (AllanRegush) [inline data.json:542]: I don't think we need this new line here. :nerd_face:

**Review by AllanRegush — COMMENTED** [review]
Looks Good :+1: Just one minor adjustment. :smile:

**Comment by AllanRegush** [comment]
Thank you for reviewing @schmelto

**Comment by ayush-09** [comment]
I have fixed the conflicts. Now check and make an approval.

**Comment by AllanRegush** [comment]
It looks there there is another conflict :nerd_face:

**Comment by ayush-09** [comment]
it's done😊

**Review by schmelto — APPROVED** [review]

**Review by Nitesh-thapliyal — APPROVED** [review]
LGTM👍

**Comment by AllanRegush** [comment]
Looks like there is a conflict. If you need any help let me know :nerd_face:

**Comment by ayush-09** [comment]
@AllanRegush  I have fixed the conflict. Please make an approval

**Comment by AllanRegush** [comment]
Thank you :+1:

**Comment by ayush-09** [comment]
@AllanRegush  Can uh tell me why my issue is closed

**Comment by eddiejaoude** [comment]
We usually close issues after the PR is merged, but we did not do that in this repo for some reason (my fault). So it was probably habit. We will go through and close all the other issues soon too for better house keeping, so we can see what actual issues need to be actioned. Do not worry your PR was merged and that is the important part 🤓

**Pull request merged**